cfme/tests/infrastructure/test_individual_host_creds.py

```diff
@@ -130,6 +133,8 @@ def test_host_bad_creds(appliance, request, setup_provider, provider, creds):
     host_collection = appliance.collections.hosts
     host_obj = host_collection.instantiate(name=test_host.name, provider=provider)
     flash_msg = msgs.get(provider.type)
+    if provider.type == 'virtualcenter':  # different message for failed remote login
```

Comment:
please replace the check for a value with a instance check for dict, that way its more generic and doesnt have a magic constant